### PR TITLE
Container Shell/Terminal

### PR DESCRIPTION
**Pull request by dogmar**

### CODE REVIEW

File: assets/src/components/terminal/TerminalInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react'
```
Comment (michaeljguarino) [inline assets/src/components/terminal/TerminalInformation.tsx:1]: Why is this file here?

**Review by maciaszczykm — APPROVED** [review]
@dogmar LGTM, feel free to merge. I'm not merging it now as there should be no conflicts with my profile changes that I'll be doing today and there is one comment from Michael.

File: assets/src/components/terminal/TerminalInformation.tsx (hunk 1 above)
Comment (dogmar) [reply assets/src/components/terminal/TerminalInformation.tsx]: Shouldn't be. It's gone.
